scipy/stats/_continuous_distns.py

```diff
@@ -3196,7 +3196,7 @@ def g(n):
         # skewness
         sk1 = _lazywhere(c >= -1./3,
                          (c, g1, g2, g3, g2mg12),
-                         lambda c, g1, g2, g3, g2gm12:
+                         lambda c, g1, g2, g3, g2mg12:
```

Comment:
how was CI passing before this?

Reply:
IIUC the last argument of the `lambda` was unused, and it  was capturing the variable from the outer scope instead. Nasty.